tests/integration/file/test_file_client.py

```diff
@@ -0,0 +1,92 @@
+# from shutil import copyfileobj
+# from tempfile import TemporaryFile
+# from typing import Optional
+
+import pytest  # type: ignore
+from nisystemlink.clients.core import ApiException
+from nisystemlink.clients.file import FileClient
+
+# from nisystemlink.clients.file.models import FileMetadata
+
+INVALID_FILE_ID = "Invalid-File-Id"
+FILE_NOT_FOUND_ERR = "Not Found"
+
+
+@pytest.fixture(scope="class")
+def client(enterprise_config) -> FileClient:
+    """Fixture to create a FileClient instance."""
+    return FileClient(enterprise_config)
+
+
+# @pytest.fixture(scope="class")
```

Comment:
We need to either get these fixtures working, or just remove them. Let's not include a bunch of commented out code.